src/components/common/ProgressBar/ProgressBar.jsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { buildStyles, CircularProgressbarWithChildren } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import { setProgressBarPathColor, setProgressBarTextColor } from '../../../utils/common/formatters/formmatters';
 import './_ProgressBar.scss';
 
 const ProgressBar = (props) => {
-  const { progress } = props;
+  const { progress, profileImg, onClickImage } = props;
```

Comment:
We can not have props specific to image in Progress Bar.
```suggestion
  const { progress } = props;
```

Reply:
done